calblueprint/siren
# login functionality

# ✨ New in this PR ✨

## One liner: what did you do? 🚨 
user authentication, login/logout with firebase and expo, basic formatting

## Coverage 🙆‍♀️
- [X] refactor navigation + add stacks

## How can the reviewer test your code? 👩‍💻
`npm start` & should automatically ask you to sign in

test credentials (feel free to add more to the firebase user database):
email: testuser@gmail.com
password: password

## Any bugs you encountered or still having trouble with? 🐛
will work on styling later when global theming is implemented

## Resources 📔
_link online resources and related PRs_ 
https://medium.com/swlh/expo-firebase-authentication-cloud-firestore-using-async-await-react-hooks-700920ad4b2b
https://blog.jscrambler.com/how-to-integrate-firebase-authentication-with-an-expo-app

🧜‍♀️ cc: @shannonbonet @phoebeli23
